dynamodb/src/main/scala/uk/gov/nationalarchives/DADynamoDBClient.scala

```diff
@@ -31,6 +31,30 @@ class DADynamoDBClient[F[_]: Async](dynamoDBClient: DynamoDbAsyncClient):
   extension [T](completableFuture: CompletableFuture[T])
     private def liftF: F[T] = Async[F].fromCompletableFuture(Async[F].pure(completableFuture))
 
+  /** Deletes a list of items with primary keys of type T from Dynamo. Unprocessed items will be retried
```

Comment:
Maybe something like `Takes a list of primary key attributes of type T and deletes the corresponding items from Dynamo...` or `Deletes items via a list of primary keys...`

Reply:
The first one is good. I've changed it.